src/main/java/io/ebeaninternal/server/query/CQuery.java

```diff
@@ -544,7 +547,7 @@ protected boolean hasNext() throws SQLException {
     return versionList;
   }
 
-  private Version readNextVersion() throws SQLException {
+  private Version<T> readNextVersion() throws SQLException {
```

Comment:
These 2 were apparently missing. All other methods involved in this change have them and the list that's being filled expects it. So I added it in.